scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1225,14 +1297,27 @@ void BPF_STRUCT_OPS(layered_running, struct task_struct *p)
 	struct cpu_ctx *cctx;
 	struct task_ctx *tctx;
 	struct layer *layer;
+	struct node_ctx *nodec;
+	struct cache_ctx *cachec;
 	s32 task_cpu = scx_bpf_task_cpu(p);
 
 	if (!(cctx = lookup_cpu_ctx(-1)) || !(tctx = lookup_task_ctx(p)) ||
 	    !(layer = lookup_layer(tctx->layer)))
 		return;
 
-	if (tctx->last_cpu >= 0 && tctx->last_cpu != task_cpu)
+	if (tctx->last_cpu >= 0 && tctx->last_cpu != task_cpu) {
+		if (!(nodec = lookup_node_ctx(cctx->node_idx)))
```

Comment:
I'm only doing the lookup for the `node_ctx`/`cache_ctx` on migration, figure it saves a few lookups.